tool_access.sh

```diff
@@ -0,0 +1,18 @@
+CLUSTER_NAME=$1
+
+CLUSTER=`./managed_kafka.sh --list | jq -r '.items[] | select(.name == "'${CLUSTER_NAME}'")'`
+
+CLUSTER_ID=`echo ${CLUSTER} | jq -r '.id'`
+
+./managed_kafka.sh --certgen ${CLUSTER_ID}
+
+SERVICE_ACCT=`grep -oP '(?<='username=\"')[^\"]*' app-services.properties`
+
+ADMIN_API_SERVER_URL=`echo ${CLUSTER} | jq -r '.admin_api_server_url'`
+
+curl -vs   -H"Authorization: Bearer $(./get_access_token.sh --owner)"   ${ADMIN_API_SERVER_URL}/api/v1/acls   -XPOST   -H'Content-type: application/json'   --data '{"resourceType":"GROUP", "resourceName":"*", "patternType":"LITERAL", "principal":"User:'${SERVICE_ACCT}'", "operation":"ALL", "permission":"ALLOW"}'
```

Comment:
Minor optimization, `./get_access_token.sh --owner` could obtain a token ahead of time and reuse it for each `curl` execution.

Reply:
okay, added that as well.